specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```
@@ -0,0 +1,3258 @@
+          "type": "string"
+        },
+        "validationStatus": {
+          "description": "Validation status of the file",
+          "enum": [
+            "NOT_VALIDATED",
+            "VALIDATION_SUCCESS",
+            "VALIDATION_FAILURE",
+            "VALIDATION_INITIATED",
+            "VALIDATION_NOT_REQUIRED"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FileStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "NOT_VALIDATED",
+                "description": "File is not validated."
+              },
+              {
+                "value": "VALIDATION_SUCCESS",
+                "description": "File is validated."
+              },
+              {
+                "value": "VALIDATION_FAILURE",
+                "description": "File validation is failed."
+              },
+              {
+                "value": "VALIDATION_INITIATED",
+                "description": "File validation is in progress."
+              },
+              {
+                "value": "VALIDATION_NOT_REQUIRED",
+                "description": "Validation is not required."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "FileInfoList": {
+      "description": "Collection of files.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of file info.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FileInfo"
+          }
+        },
+        "nextLink": {
+          "description": "Link for the next list of file URLs, if applicable",
+          "type": "string"
+        }
+      }
+    },
+    "LoadTestConfiguration": {
+      "description": "The load test configuration.",
+      "type": "object",
+      "properties": {
+        "engineInstances": {
+          "format": "int32",
+          "description": "The number of engine instances to execute load test. Supported values are in range of 1-45. Required for creating a new test.",
+          "type": "integer"
+        },
+        "splitAllCSVs": {
+          "description": "If false, Azure Load Testing copies and processes your input files unmodified across all test engine instances. If true, Azure Load Testing splits the CSV input data evenly across all engine instances. If you provide multiple CSV files, each file will be split evenly.",
+          "default": false,
+          "type": "boolean"
+        },
+        "quickStartTest": {
+          "description": "If true, optionalLoadTestConfig is required and JMX script for the load test is not required to upload.",
+          "default": false,
+          "type": "boolean"
+        },
+        "optionalLoadTestConfig": {
+          "$ref": "#/definitions/OptionalLoadTestConfig"
+        }
+      }
+    },
+    "Metric": {
```

This schema is the response body for "TestRun_GetMetrics" so I think the name should be plural to match the operation name.
```suggestion
    "Metrics": {
```

---

changed